lib/archives_sidebar/lib/archives_sidebar.rb

```diff
@@ -30,11 +30,11 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%13)
```

Comment:
fixes #2 : when you remove the +1, the remainder of entry.month.to_i is 0 and messes up the later logic. Therefore you need to change it to %13 to maintain the remainder.